templates/education/pdce-workshops.php

```diff
@@ -46,15 +65,15 @@
 
 			<div class="boxey-inner">
 
-				<a href="#modal_<?php echo absint( $post_id ); ?>" target="_blank" class="fancybox-inline"><span class="iso-title"><?php echo esc_html( $the_title ); ?></span></a>
+				<a href="<?php echo esc_html( $permalink ); ?>" target="_blank"><span class="iso-title"><?php echo esc_html( $the_title ); ?></span></a>
```

Comment:
Will it always be a URL? i.e. not some string like in the modal? In that case `esc_url()` is more appropriate here, I think?